api/communication-api/src/main/java/com/cognifide/aet/communication/api/metadata/Test.java

```diff
@@ -38,7 +39,7 @@
 
   @Valid
   @NotNull(message = "Test must have at least one url")
-  private final Set<Url> urls = new HashSet<>();
+  private Set<Url> urls = new HashSet<>();
```

Comment:
Why `urls` is no longer `final`?

Reply:
If this is for unit tests only, maybe it would be sufficient to make this setter package-protected?

Reply:
I can not make it package-protected because it is not for the test this class but for the class that used Test class